examples/workflows/example_gaussian_spin.ipynb

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
It is not clear to me why do you prefer a jupyter notebook here. I think a simple file would be preferable, as it is doing the same and additionally, can be used to run a test calculation.

Reply:
Well here we have to take a decision whether to have or not the possibility to include in the distribution also useful jupyter notebooks that help testing. The workchian produces results that can be easily checked in a notebook while a bare py execution would require more time for the analysis of results I guess.
Back to testing: I guess we have no chance for automatic testing of gaussian due to licensing problems

Reply:
Sasha, I agree that in principle the submission and running can be done with a normal python script. If the post-process output was only a string, then even that could be included in the script. BUT the post-processing includes images in output, which can only be nicely shown in the notebook.

Regarding testing: we should implement these details when we're working on that. I think at this stage it doesn't make sense to try to do everything within this PR. When we have a nice starting point that demonstrates the usage and post-process, then it's easy in the future to turn it into a script or anything else we decide.

Reply:
My point is that this repository is mostly about workflows/plugins/parsers. I consider it as a set of nice tools that are useful for computational scientists. The visualisation of the data should go to the aiidalab applications, I believe.